test/vagrant-service-manager/installer_test.rb

```diff
@@ -71,6 +71,17 @@ module ServiceManager
           @installer.instance_variable_get('@binary_handler').binary_exists.must_equal true
         end
       end
+
+      describe 'Kubernetes' do
+        before do
+          @type = :kubernetes
+          @options = { box_version: 'adb' }
+        end
+
+        it 'should displays unsupported message' do
+          assert_raises(SystemExit) { @installer = Installer.new(@type, @machine, @machine.env, @options) }
```

Comment:
We can only verify exit at the moment. Don't know how to verify message. 
Updated the test case description